package.json

```diff
@@ -17,6 +17,7 @@
     "@fortawesome/react-fontawesome": "^0.0.17",
     "classnames": "^2.2.5",
     "firebase": "^4.9.0",
+    "lodash": "^4.17.5",
```

Comment:
Do we really need this? Could we copy paste what functionality we do need instead?

If we do need it, please measure main.js size and report in PR so we can add it to https://github.com/rahafoundation/raha.io/issues/45 after land.